aten/src/ATen/native/mkl/SpectralOps.cpp

```diff
@@ -479,6 +479,16 @@ static Tensor& _exec_fft(Tensor& out, const Tensor& self, IntArrayRef out_sizes,
   const auto value_type = c10::toRealValueType(input.scalar_type());
   out.resize_(batched_out_sizes, MemoryFormat::Contiguous);
 
+  // fix mkl issue
+  // https://github.com/pytorch/pytorch/issues/154477
```

Comment:
As one is linking with MKL dynamically, don't you want to guard this check against `mkl_getmajor()> 11` or something of the nature?